Makefile

```diff
@@ -127,6 +127,7 @@ DUNESTRAPOPT=--root .
 # We regenerate always as to correctly track deps, can do better
 # We do a single call to dune as to avoid races and locking
 _build/default/theories_dune _build/default/ltac2_dune .dune-stamp: FORCE
+	$(if $(COQ_DUNE_EXTRA_OPT),echo '()' > dunestrap.inc)
```

Comment:
not sure this is the best way to work, basically in split mode we shouldn't be depending on the local coq-core META

Reply:
Why not? Note that the generation of the `META` file is (or was) O(1) and doesn't build anything else.

Reply:
When we use `-p coq-stdlib` dune says "no rule for ..." for stuff in coq-core